### PR TITLE
Copy ExaPF files

### PR DESCRIPTION
**Pull request by dmaldona**
Copy main ExaPF files sans LinearAlgebra and AutoDiff (these will be different packages).

Adding ExaPF as a dependency during refactoring.

### CODE REVIEW

File: src/Aesop.jl [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+using SparseArrays
+
+using KernelAbstractions
+const KA = KernelAbstractions
+
+include("PowerSystem/PowerSystem.jl")
+using .PowerSystem
+const PS = PowerSystem
+
+include("templates.jl")
```
Comment (michel2323) [inline src/Aesop.jl:15]: I think the abstract types can go directly into Aesop.jl.

File: src/Aesop.jl [hunk 2]
```diff
@@ -1,5 +1,20 @@
 module Aesop
 
-greet() = print("Hello World!")
+import ExaPF: AutoDiff # temporary
```
Comment (michel2323) [inline src/Aesop.jl:3]: This would be ADONIS, right?

**Review by michel2323 — APPROVED** [review]
This looks good to me. Time to open issues. Let's go!

File: src/Aesop.jl (hunk 2 above)
Comment (dmaldona) [reply src/Aesop.jl:3]: @michel2323 right!

**Pull request merged**